digdag-core/src/main/java/io/digdag/core/log/LocalFileLogServerFactory.java

```diff
@@ -207,13 +215,19 @@ public void log(byte[] data, int off, int len)
             public void close()
             {
                 try {
-                    temporaryOutput.close();
-
-                    try (InputStream in = Files.newInputStream(temporaryPath);
-                         OutputStream conclusiveOutput = new GZIPOutputStream(Files.newOutputStream(conclusivePath, CREATE, APPEND), 16*1024)) {
-                        ByteStreams.copy(in, conclusiveOutput);
-                        conclusiveOutput.close();
-                        temporaryPath.toFile().delete();
+                    output.close();
+
+                    if (realtimeFollow) {
+                        // Delaying for deterrence duplicate output in digdag log --follow
+                        try { Thread.sleep(1000); }
```

Comment:
Same as above < code format

Where did this 1 sec come from?
Also, this can affect the performance of enqueuing tasks of io.digdag.core.agent.MultiThreadAgent#run...

Reply:
This sleep for duplicate output in `digdag log --follow`.
If do not this, output will duplicate such as

```
$ digdag log 1 --follow
1 (from plain text)
2 (from plain text)
3 (from plain text)
4 (from plain text)
5 (from plain text)
(finish the workflow, plan text file will be removed, and gzip file will be created)
1 (from gzip all of below)
2
3
4
5
```

I think this way is not well, can we solve the issue another way?